modules/xmpp/SdpConsistency.js

```diff
@@ -0,0 +1,176 @@
+import * as transform from 'sdp-transform';
+
+/**
+ * Begin helper functions
+ */
+/**
+ * Given a video mline (as parsed from transform.parse),
+ *  return the single primary video ssrcs
+ * @param {object} videoMLine the video MLine from which to extract the
+ *  primary video ssrc
+ * @returns {number} the primary video ssrc
+ */
+function getPrimarySsrc (videoMLine) {
+    if (!videoMLine.ssrcs) {
+        return;
+    }
+    let numSsrcs = videoMLine.ssrcs
+        .filter(ssrcInfo => ssrcInfo.attribute === "msid")
+        .map(ssrcInfo => ssrcInfo.id)
+        .filter((ssrc, index, array) => array.indexOf(ssrc) === index)
+        .length;
+    if (numSsrcs === 1) {
+        return videoMLine.ssrcs
```

Comment:
i think we can make this even simpler actually, if there is only 1 ssrc we can just do:

  return videoMLine.ssrcs[0].id;